### PR TITLE
Section-by-Section header fix

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Fixed a bug with the section-by-section analysis.
 
## Additions
 
- There's some differences with the way grunt compiles some of the JavaScript as a result of #682.
 
## Removals
 
- Removed a fix to the sxs header that was breaking things now. I checked this in older browsers to make sure it wasn't necessary anymore.
 
## Testing

Unfortunately, our testing API doesn't have data for the Section-by-section analysis, so I had to manually check these in a VM.
 
## Review
 
- @ascott1 
 
## Preview
 
**Before**
old sxs

**After**
new sxs

### CODE REVIEW

File: regulations-core [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 9c8324d26ab8291f9d7ac0898ab707f076199c43
```
Comment (ascott1) [inline regulations-core]: @KimberlyMunoz Let's remove this as it's from the testing API. I *think* it should be in the .gitignore now, so it shouldn't be a problem moving forward.

**Comment by ascott1** [comment]
Good catch! I added the note above, but once that's out this is good to merge.

File: regulations-core (hunk 1 above)
Comment (KimberlyMunoz) [inline regulations-core]: Removed!

**Comment by ascott1** [comment]
:+1: Thanks!

**Pull request merged**